build/bakefiles/multilib.bkl

```diff
@@ -180,6 +180,14 @@
         <if cond="TOOLKIT=='MSW'">$(TOP_SRCDIR)3rdparty/webview2/build/native/include</if>
     </set>
 
+    <set var="webview_additional_include_wrl">
+        <if cond="TOOLKIT=='MSW' and IS_MSVC=='0'">$(TOP_SRCDIR)include/wx/msw/wrl</if>
+    </set>
+
+    <set var="webview_edge_pragma_warning">
+        <if cond="TOOLKIT=='MSW' and IS_MSVC=='0'">-Wno-unknown-pragmas</if>
```

Comment:
I don't have any problems with doing it like this, but using `pragma GCC diagnostic push/ignored/pop` before/after the code containing the unknown pragmas could have been an even simpler solution.

Reply:
That is what I did initially, see https://github.com/wxWidgets/wxWidgets/commit/e67b0af7601a3d49b61636034cc6d9486a01ea9f#diff-3c03e1f5645f42320bb127a440d48aa10b9c1f70179df0a3f316fe022abc71f6

But this did not work for GCC, the warning still shows.

Reply:
Sorry for missing this, I now see PB has even found the corresponding gcc bug already, but I didn't realize its import.